client/my-sites/stats/all-time-highlights-section/post-cards-group.tsx

```diff
@@ -53,40 +54,41 @@ const getStatsQueries = createSelector(
 );
 
 const getStatsData = createSelector(
-	( state, siteId, topPostsQuery ) => {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	( state, siteId, topPostsQuery, isTopViewedPostRequesting ) => {
```

Comment:
```suggestion
	( state, siteId, topPostsQuery ) => {
```

It should be fine to exclude `isTopViewedPostRequesting` from the selector function since we're not using it.

`isTopViewedPostRequesting` will still be available for the cache-busting function since `getStatsData` is invoked with it as its fourth parameter on line 91.

Reply:
Yeah, I do definitely want to exclude it. In fact, it is [intended to be left](https://github.com/Automattic/wp-calypso/pull/74495/commits/5896bf076be7a6cde8c0ff50fe54cbab2778bd9a) to avoid another warning for the second argument that would break the unit tests. Would you happen to have any idea about the warning? @jsnmoon

<img width="1203" alt="截圖 2023-03-18 上午12 08 46" src="https://user-images.githubusercontent.com/6869813/225959638-da78d191-0a5c-4127-96e1-36b379eae133.png">

Reply:
I see; that's undesirable.

It'd be great to fix this, but fixing the typing error is more trouble than it's worth. Let's ignore this for now.